e2e/test/vue-javascript/package-lock.json

```diff
@@ -11185,8 +11185,7 @@
             "console-control-strings": {
```

Comment:
I don't quite like that there are changes in other packages - it should be Angular E2E change - tho i guess it was done automatically so 🤷

Reply:
You're right, I missed this.